src/app/home-page/home-page.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { fakeVideoList } from '../core/mocks/mocked-data';
+import { CourseItemInfo } from '../core/models';
+import { OrderByTitleNamePipe } from '../core/pipes/orderByTitleName.pipe';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class HomePageService {
+  private coursesList: CourseItemInfo[] = fakeVideoList;
```

Comment:
storing data in services isn't a very good idea. in some cases, it might be necessary but not here.